nuxeo-services/nuxeo-invite/src/main/java/org/nuxeo/ecm/user/invite/UserInvitationComponent.java

```diff
@@ -574,7 +575,7 @@ public DocumentModelList getRegistrationsForUser(final String docId, final Strin
             public void run() {
                 String query = "SELECT * FROM Document WHERE ecm:currentLifeCycleState != 'validated' AND"
                         + " ecm:mixinType = '" + getConfiguration(configurationName).getRequestDocType()
-                        + "' AND docinfo:documentId = '%s' AND"
+                        + "' AND docinfo:documentId = '%s' AND "
```

Comment:
While you are modifying on the query, it would be interesting to rework it to use only `%s` instead of a mix of concatenation and `%s`.

Reply:
yes, I agree, i was wondering about it, if i should replace concatenation by '%s' or at least using Stringbuilder. But like,  there is no way to do test (i spent time to find a way :( without success) and the need of backport deterred me. But as you mentioned we can use a fake docinfo.xsd in T.U but what if the original docinfo.xsd changes

Reply:
Stick to `%s` and use `String.format` as it's already here.
The missing space in the query is quite obvious to not need to create fake test schema..., seems that @kevinleturc agreed.